api/server/objectstore.go

```diff
@@ -0,0 +1,175 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/libopenstorage/openstorage/objectstore"
+)
+
+// swagger:operation GET /cluster/objectstore objectstore objectStoreInspect
+//
+// Lists Objectstore
+//
+// This will list current object stores
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - name: ID
+//   in: query
+//   description: ID of objectstore to inspect
+//   required: true
+//   type: string
+// responses:
+//   '200':
+//     description: success
+//     schema:
+//      $ref: '#/definitions/ObjectstoreInfo'
+func (c *clusterApi) objectStoreInspect(w http.ResponseWriter, r *http.Request) {
+	method := "objectStoreInspect"
+	params := r.URL.Query()
+	objstoreID := params[objectstore.ObjectStoreID]
+
+	if len(objstoreID) == 0 || objstoreID[0] == "" {
```

Comment:
This should be optional. If there is only one objectstore configured there shouldn't be a need to pass in the objectstore ID. Better to leave this up to the implementer of ObjectStoreInspect()